uefi-pi/src/pi/guid.rs

```diff
@@ -0,0 +1,95 @@
+use core::{convert::TryInto, mem::size_of, ptr::slice_from_raw_parts};
```

Comment:
It would be better to add license header too:)
And it a PR against the main branch is needed for this.